### PR TITLE
cli-output: Add CliTokenAccount type

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem

No support for SPL Token accounts in `cli-output` crate

#### Summary of Changes

Add it

### CODE REVIEW

File: cli-output/src/cli_output.rs [hunk 1]
```diff
@@ -1149,6 +1150,51 @@ impl fmt::Display for CliFees {
...
+    pub token_account: UiTokenAccount,
+}
+
+impl fmt::Display for CliTokenAccount {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        writeln!(f)?;
+        writeln_name_value(f, "Address:", &self.address)?;
+        let account = &self.token_account;
+        println!("decimals: {}", account.token_amount.decimals);
+        println!("amount: {}", account.token_amount.amount);
```
Comment (CriesofCarrots) [inline cli-output/src/cli_output.rs:1167]: Shouldn't be `println`; but maybe remove these lines altogether?

File: cli-output/src/cli_output.rs [hunk 2]
```diff
@@ -1149,6 +1150,51 @@ impl fmt::Display for CliFees {
...
+        writeln_name_value(f, "Owner:", &account.owner)?;
+        writeln_name_value(f, "State:", &format!("{:?}", account.state))?;
+        if let Some(delegate) = &account.delegate {
+            writeln!(f, "Delegation:")?;
+            writeln_name_value(f, "  Delegate:", delegate)?;
+            let allowance = account.delegated_amount.as_ref().unwrap();
+            writeln_name_value(f, "  Allowance:", &allowance.real_number_string_trimmed())?;
+        } else {
+            writeln_name_value(f, "Delegation:", "")?;
+        }
```
Comment (CriesofCarrots) [inline cli-output/src/cli_output.rs:1188]: Skip this?

File: cli-output/src/cli_output.rs (hunk 1 above)
Comment (t-nelson) [reply cli-output/src/cli_output.rs:1167]: Oops!  Yep these were for debug 😅

File: cli-output/src/cli_output.rs (hunk 2 above)
Comment (t-nelson) [reply cli-output/src/cli_output.rs:1188]: Sure!

**Comment by t-nelson** [comment]
Both done!

**Review by CriesofCarrots — APPROVED** [review]

**Pull request merged**